index.html

```diff
@@ -69,9 +81,7 @@ <h2>Tonic</h2>
                 </ul>
             </div>
             <div class="button">
-                <p>
-                    See Project
-                </p>
+                <button type="button" class="work-button">See Project</button>

```

Comment:
- [ ] As suggested by the previous reviewer, kindly make the hover effect on all the buttons and style be like the Figma version. Kindly make sure that when you hover on to any button in the project the color changes as the design.

![2022-06-02](https://user-images.githubusercontent.com/70269507/171627930-ba5eb482-8aff-4025-8587-b95c8b38a52a.png)